akka-http-core/src/main/mima-filters/10.1.11.backwards.excludes/internal-RenderSupport-changes.excludes

```diff
@@ -0,0 +1,2 @@
+# Changes to internal methods
```

Comment:
should this be `10.1.x.backwards.excludes` or do we intend to backport?

Reply:
No, I don't think we should backport there. I'll fix it.